### PR TITLE
Add Jack workaround code

### PR DESCRIPTION
**Pull request by a7r3**
Adapted from

https://github.com/TeamVee-SS/android_.repo_local_manifests/commit/ceddb0db6c424f3b0c54f05d89b417586fa17475

Signed-off-by: Arvindraj <arvindultimate7352@gmail.com>

### CODE REVIEW

**Comment by a7r3** [comment]
Shellcheck later

**Comment by a7r3** [comment]
If there's something wrong in the authorships (or signoffs) lemme know @Caio99BR 

Travis && ShellCheck can be solved, leave it with me

**Comment by Caio99BR** [comment]
No, problem with autorship or signoff.
How can I test it?

**Comment by a7r3** [comment]
No significant changes were made, I can explain what I changed in the comments

File: ROM.sh [hunk 1]
```diff
@@ -1221,6 +1221,37 @@ function build() # 4
                         ;;
                     *) echo -e "${FLD} Invalid Selection.\n" ;;
                 esac
+                # Jack cannot be disabled in N
+                # Jack workaround prompt is asked if this is set to y/Y
+                SBJK="y";
+            fi
+            if [[ "${SBJK}" == [Yy] ]] && [[ "$(free | awk '/^Mem:/{print $2}')" -lt 4096 ]]; then
```
Comment (a7r3) [inline ROM.sh]: `free -m` didn't work
```[arvind@AspiringManjaro ~]$ free -m
free: Multiple unit options doesn't make sense.
```

`free` works properly

`-lt` should be used for comparing integers

`${2}` -> `$2` - `awk` threw an error on that

**Comment by a7r3** [comment]
Travis-shellcheck passed :smiley:

**Pull request merged**